src/features/app/components/Navbar/UserOptions.test.js

```diff
@@ -0,0 +1,80 @@
+import { fireEvent, render } from 'testUtils';
+
+import { userData, authenticationHeaders } from 'testUtils/mocks/auth';
+import UserOptions from 'features/app/components/Navbar/UserOptions';
+
+const state = {
+  auth: {
+    user: userData,
+    session: authenticationHeaders,
+  },
+};
+
+describe('UserOptions', () => {
+  it('renders an avatar when the user is logged in', () => {
+    const { getByRole } = render(<UserOptions />, { state });
+
+    getByRole('img', { name: 'avatar' });
+  });
+
+  it('does not render an avatar when the user is not logged in', () => {
+    const { queryByTestId } = render(<UserOptions />);
+
+    expect(queryByTestId('avatar')).toBeNull();
+  });
+
+  it('opens the menu if the user clicks on the avatar', () => {
+    const { getByRole } = render(<UserOptions />, { state });
+
+    const avatar = getByRole('button', { name: 'avatar' });
+
+    fireEvent.click(avatar);
+
+    getByRole('button', { name: 'Sign out' });
+  });
+
+  it('opens the menu when the user is not logged in and clicks on the menu button', () => {
+    const { getByRole } = render(<UserOptions />);
+    const menuButton = getByRole('button');
+
+    fireEvent.click(menuButton);
+
+    getByRole('button', { name: 'Sign in' });
+  });
+
+  it('closes the menu when the user is logged in and clicks again', () => {
```

Comment:
I think we shouldn't test thing here because it's too specific and it's more related to the useClickAway hook. maybe we could test the useClickAway hook separately instead

Reply:
FYI: I'm not saying you should add tests for it on this PR